modules/terraform/cmd.go

```diff
@@ -59,3 +65,35 @@ func RunTerraformCommandE(t *testing.T, options *Options, args ...string) (strin
 		return out, retry.FatalError{Underlying: err}
 	})
 }
+
+// GetExitCodeForTerraformCommand runs terraform with the given arguments and options and returns exit code
+func GetExitCodeForTerraformCommand(t *testing.T, additionalOptions *Options, args ...string) int {
+	exitCode, err := GetExitCodeForTerraformCommandE(t, additionalOptions, args...)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return exitCode
+}
+
+// GetExitCodeForTerraformCommandE runs terraform with the given arguments and options and returns exit code
+func GetExitCodeForTerraformCommandE(t *testing.T, additionalOptions *Options, args ...string) (int, error) {
+	options := GetCommonOptions(additionalOptions, args...)
+
+	logger.Log(t, "Running terraform %v", args)
+	cmd := shell.Command{
+		Command:    "terraform",
+		Args:       args,
+		WorkingDir: options.TerraformDir,
+		Env:        options.EnvVars,
+	}
+
+	_, err := shell.RunCommandAndGetOutputE(t, cmd)
+	if err == nil {
+		return TerraformPlanNoChangeExitCode, nil
```

Comment:
Seems like this isn't the best variable to use here, since this is a generic `GetExitCodeForTerraformCommandE` method, and has nothing to do with the `plan` command. Perhaps add one more constant for use here called `DefaultSuccessExitCode`? It's also set to 0, but it'll communicate the intent a bit more clearly.

Reply:
done